new-pipeline/src/evo.js

```diff
@@ -877,10 +859,14 @@ function displayEvolution(target, lines, usePercentages, plotOptions) {
 
 }
 
-function displayEvolutions(lines, submissionLines, useSubmissionDate,
-  usePercentages) {
+function displayEvolutions(lines, useSubmissionDate, usePercentages) {
   indicate("Rendering evolutions...");
 
+  var submissionLines = lines.filter(line => line.aggregate == "Submissions");
+
+  let metaAggregateNames = gMetaAggregates.map(entry => entry[1]);
+  lines = lines.filter(line => !metaAggregateNames.includes(line.aggregate));
```

Comment:
This imposes a constraint that aggregates and metaAggregates cannot share the same name. Not likely to come up, but you should put it in a comment up near gMetaAggregates.